pandas/core/indexes/multi.py

```diff
@@ -945,6 +951,34 @@ def get_level_values(self, level):
         values = self._get_level_values(level)
         return values
 
+    def unique(self, level=None):
```

Comment:
I would add the ``level=None`` kw to Index as well (for compat. It should raise if its not None).

Reply:
OK, would you mention it in the (``Index``) docs?